packages/Bio/src/tests/sequence-space-test.ts

```diff
@@ -14,7 +14,9 @@ category('sequenceSpace', async () => {
   let testHelmWithEmptyRowsTableView: DG.TableView;
 
   test('sequenceSpaceOpens', async () => {
-    testFastaDf = await readDataframe(DG.Test.isInBenchmark ? 'test/peptides_motif-with-random_10000.csv':'tests/100_3_clustests.csv');
+    testFastaDf = await readDataframe(
+      DG.Test.isInBenchmark ? 'test/peptides_motif-with-random_10000.csv':'tests/100_3_clustests.csv'
```

Comment:
All operators should have spaces before and after (`:` in this case).
Also, this should be rewritten as follows:
```
testFastaDf = await readDataframe(DG.Test.isInBenchmark ? 'test/peptides_motif-with-random_10000.csv' :
  'tests/100_3_clustests.csv');
 ```
 Or alternatively, if it fits the line length limit:
 ```
testFastaDf = await readDataframe(
  DG.Test.isInBenchmark ? 'test/peptides_motif-with-random_10000.csv' : 'tests/100_3_clustests.csv');
 ```

Reply:
Done, also Added rule for such cases.